frontend/language/src/nb.json

```diff
@@ -49,6 +49,7 @@
   "app_data_modelling.properties_information": "Ingen egenskaper vises for øyeblikket fordi ingen elementer er valgt. Velg et element for å konfigurere og vise egenskapene til det valgte elementet.",
   "app_data_modelling.select_xsd": "Velg en XSD",
   "app_data_modelling.upload_xsd": "Last opp",
+  "app_data_modelling.upload_xsd_invalid_error": "Datamodellnavnet er ugyldig, sjekk om du har brukt ugyldige tegn i navnet eller lag et nytt navn.",
```

Comment:
Do we inform the user about what is valid name or what is invalid characters? 😊

Reply:
Good question 😊 based on the validation, It is allowed to use upper/lowercase, digit, underscore, period , hyphen, and Norwegian characters. Do we reformulate the message to be more specific for these allowed things, maybe?

Reply:
Yes, sounds like you should ask @Ildest about this one. 😊